webaverse-studios/app
# Solana mint

## Describe your changes
This PR is for integrating Webaverse and Solana.
Users can login with Phantom wallet and mint their assests on the Solana by drag and drop.
I forked https://github.com/webaverse/app/tree/hedera-voucher_sign branch and added Solana mint function to it.

## What are the steps for a QA tester to test this pull request?
1. Login with phantom wallet.
2. Drag and drop assets such as .jpg, .glb or .vrm files on the webaverse app.
3. Enter the NFT name and details in the mint modal and click the "Mint on solana" button to mint the NFT.
4. Check the minted NFT in your inventory and solscan.io.

## Issue ticket number and link

## Screenshots and/or video

## Checklist before requesting a review
- [ ] I have completed the entire [QA checklist](https://github.com/webaverse/app/issues/3153) on my PR

https://user-images.githubusercontent.com/93294276/192174055-a84bc88d-99e1-4e5d-9151-a7f63a34da7f.mp4
